tests/cucumber/steps/steps.js

```diff
@@ -4284,8 +4292,10 @@ module.exports = function getSteps(options) {
         appAccounts,
         appForeignApps: foreignApps,
         appForeignAssets: foreignAssets,
-        type: 'appl',
-        suggestedParams: sp,
+        note: undefined,
+        lease: undefined,
+        reKeyTo: undefined,
```

Comment:
Must these be explicitly set to `undefined`? Does the test fail when these properties are not set?

Reply:
In this case it may work because its from an object, but removing these parameters gives issues in other tests so I rather keep it consistent

Reply:
I personally disagree - I think it's generally bad practice to set undefined properties on an object, but I'll wait to see what @jasonpaulos thinks.

Reply:
In this case I'd say the `undefined` fields should be omitted since they have nothing to do with this test.

Reply:
cool thanks yall